pkg/controller.v1/xgboost/xgboost.go

```diff
@@ -93,7 +93,7 @@ func SetPodEnv(job interface{}, podTemplate *corev1.PodTemplateSpec, rtype, inde
 		})
 		podTemplate.Spec.Containers[i].Env = append(podTemplate.Spec.Containers[i].Env, corev1.EnvVar{
 			Name:  "PYTHONUNBUFFERED",
-			Value: "0",
+			Value: "1",
```

Comment:
Please revert this change. We can work on this in a separate PR (I would suggest we make changes for both PaddleJob and XGboostJob in a separate same PR).